src/renderers/webgl/WebGLTextures.js

```diff
@@ -1593,7 +1593,7 @@ function WebGLTextures( _gl, extensions, state, properties, capabilities, utils,
 
 					const glFormat = utils.convert( texture.format, texture.encoding );
 					const glType = utils.convert( texture.type );
-					const glInternalFormat = getInternalFormat( texture.internalFormat, glFormat, glType, texture.encoding );
+					const glInternalFormat = getInternalFormat( texture.internalFormat, glFormat, glType, texture.encoding, true );
```

Comment:
Would you mind renaming the `isVideoTexture` parameter of `getInternalFormat()` to `forceLinearEncoding`?

I think this would make the code easier to read. Right now, it seems strange that the above code use a flag intended for video textures.

Reply:
I renamed it to `forcePassthroughEncoding` because if the incoming format wasn't linear, this function is not going to change it to it.

Reply:
Just for my understanding: How does this change affect the non-VR usage of multisampled render targets? Is it still possible to save the contents of a multisampled render target as sRGB? Or the data now always linear color space?

Reply:
Without this change, the multisampled renderbuffer would be allocated as sRGB while the texture is in RGB.
This will not only result in incorrect rendering (since you now render to sRGB), the blitframebuffer call will also fail because the color spaces don't match.
So, if this pass was broken and this change shouldn't affect content that was still working.